client/components/payments-status/style.scss

```diff
@@ -0,0 +1,17 @@
+@import '../../styles/abstracts/colors';
+.account-details__info {
```

Comment:
(minor) Since this is in the `payments-status` component, the class name should also try to match for consistency